src/components/dashboard/dashboard.tmpl.html

```diff
@@ -37,42 +37,42 @@
 
   <!-- Widgets selection container -->
   <div id="widget-selector" collapse="!showWidgetSelector()" ng-if="!customWidgetSelector.path">
-    <div class="title">
-      <p>Select the widgets you want to add to your dashboard.</p>
-      <span class="badge confirmation">Widget added!</span>
-      <i class="fa fa-times-circle" ng-if="showCloseWidgetSelectorButton()" ng-click="displayWidgetSelector(false)"/>
+    <div class="widget-selector__title">
```

Comment:
Wouldn't it be preferable to keep the naming consistent with the rest of the lib?
```less
.widget-selector {
  .title {
    .instruction { }
  }
}
```